tests/integration_test.rs

```diff
@@ -469,3 +472,39 @@ fn test_uses_expected_gas() {
     assert_eq!(result, U256::from_dec_str("3").unwrap());
     assert_eq!(final_vm_state.gas_left(), 0_u32);
 }
+
+#[test]
+fn test_sload_with_present_key() {
+    let bin_path = make_bin_path_asm("sload_key_present");
+    let (result, _) = run_program_in_memory(&bin_path);
+    assert_eq!(result, U256::from_dec_str("3").unwrap());
+}
+
+#[test]
+fn test_sload_with_absent_key() {
+    let bin_path = make_bin_path_asm("sload_key_absent");
+    let (result, _) = run_program_in_memory(&bin_path);
+    assert_eq!(result, U256::zero());
+}
+
+#[test]
+fn test_tload_with_present_key() {
+    let bin_path = make_bin_path_asm("tload_key_present");
+    let (result, _) = run_program_in_memory(&bin_path);
+    assert_eq!(result, U256::from_dec_str("3").unwrap());
+}
+
+#[test]
+fn test_tload_with_absent_key() {
+    let bin_path = make_bin_path_asm("tload_key_absent");
+    let (result, _) = run_program_in_memory(&bin_path);
+    assert_eq!(result, U256::zero());
+}
+
+// TODO: All the tests above should ran with this storage as well.
```

Comment:
Okay, that's actually not a bad idea. Talking with @jrchatruc a few days ago, we decided that it makes more sense right now to test everything with the in-memory storage. This approach helps avoid potential problems related to database creation and possible weird states with the database not deleting itself after a failed test. So, for the current state of the VM, it's better to avoid this, but it will be useful in the future.